### ISSUE TITLE
Feature: Support eventMouseEnter & eventMouseLeave

### ISSUE THREAD

**Issue by eiswind** [stefanuebe/vaadin_fullcalendar#147]
In our project we would like to use the eventMouseEnter/eventMouseLeave events from fullcalendar. 
We created a simple working solution by adding:

```
_createEventHandlers() {
.....
.....
eventMouseEnter: (eventInfo) => {
  let event = eventInfo.event;
  return {
        id: event.id, 
        data: this._toEventData(event)
    }
},
eventMouseLeave: (eventInfo) => {
    let event = eventInfo.event;
    return {
        id: event.id, 
        data: this._toEventData(event)
    } },
```

We would appreciate if you could support this in one of the next releases. If you agree, I would create a PR as the next step.

**Comment by aetasoul** [comment]
Hi,
Thanks for the suggestion!

I think we'll implement in the next release without problem!

Feel free to open the PR and I'll check it :)

Refs for myself https://fullcalendar.io/docs/eventMouseEnter

**Comment by eiswind** [comment]
Thanks for the quick reply. I created
https://github.com/stefanuebe/vaadin_fullcalendar/pull/148

**Comment by aetasoul** [comment]
Hi @eiswind 

You can now use `calendar.addEntryMouseEnterListener` and `calendar.addEntryMouseLeaveListener`.

We'll release it with the v4.2, if you need it earlier the changes were made on the master branch

**Comment by eiswind** [comment]
Thanks a lot for merging this so quickly.

**Comment by stefanuebe** [comment]
Released with 4.1.3.
